specs/rm/requirements-management-spec.html

```diff
@@ -33,8 +33,8 @@
         shortName: "requirements-management-spec",
         specStatus: status,
         revision: revision.toString().padStart(2, "0"),
-        publishDate: "2019-10-01",
-        previousPublishDate: "2018-05-31",
+        publishDate: "2020-xx-xx",
+        previousPublishDate: "2018-08-24",
```

Comment:
The date for https://docs.oasis-open-projects.org/oslc-op/rm/v2.1/psd02/ is 01-Oct-2019. Where did you get yours?

Reply:
http://docs.oasis-open.org/oslc-domains/oslc-rm/v2.1/oslc-rm-v2.1-part1-requirements-management-spec.html
That's why I thought previousPublishDate is "2018-08-24"-